app/controllers/works_controller.rb

```diff
@@ -92,6 +92,8 @@ def work_form(work)
   sig { params(work: Work).void }
   def after_save(work)
     work.event_context = { user: current_user }
+    work.event_context[:description] = params[:work][:version_description] unless params[:work][:version_description].nil?
```

Comment:
Would you consider:

```suggestion
work.event_context = { user: current_user, description: params.dig(:work, :version_description) }.compact
```

:question:

Reply:
I like.